### ISSUE TITLE
Don't disable UI mods

### ISSUE THREAD

**Issue by vulcan-dev** [BeamMP/BeamMP#206]
**Describe the solution you'd like**
When joining a server, all your mods get disabled but I believe that **UI** mods should be an exception. I don't like the default UI elements for BeamNG so using modded ones in BeamMP would be great.

**Comment by 5TXdYWK** [comment]
Is this really BeamMP's responsibility? Re-enable the mods after loading in or ask the server owner to add them to the server's modlist?

**Comment by daniel-w0** [comment]
> 

It's called user experience, it's an easy thing for them to add. Also, why should server owners add the mods for it? Makes no sense to have people download unnecessary mods.

**Comment by lionkor** [comment]
I actually agree this should be a thing! Is there a way to detect that a mod is *only* doing UI?

**Comment by daniel-w0** [comment]
> 

Yes, you can get mods via `core_modmanager.getModDB(path)` and then with that, you have access to `modType` which in this case would be `app`. Here's how I done it for [KissMP](https://github.com/vulcan-dev/KISS-multiplayer/commit/bbf4ad61cbdc4f36e079af03544a9e2bfeca9338)

**Comment by lionkor** [comment]
Sounds good, thank you!

We have to make this toggleable by servers, though, since for some servers UI mods can break the experience.

**Comment by daniel-w0** [comment]
Alright, perfect. Thanks 👍
